disnake/widget.py

```diff
@@ -389,9 +399,12 @@ async def fetch_invite(self, *, with_counts: bool = True) -> Invite:
 
```

Comment:
this one is missing a versionchanged for sure

Reply:
be1c601932eadc9d4146715fa63dd5e4e978d385